yarn.client.failover-retries-on-socket-timeouts	0	ERROR	Thread Thread[eventHandlingThread,5,main] threw an Exception.java.lang.NegativeArraySizeException: -1130426860
		30